test.disabled/ejabberd_tests/tests/push_pubsub_SUITE.erl

```diff
@@ -371,10 +369,13 @@ bare_jid(JIDOrClient) ->
 
 %% ----------------------------------------------
 %% REST mock handler
-setup_mock_rest(Port) ->
+setup_mock_rest() ->
     TestPid = self(),
     HandleFun = fun(Req) -> handle(Req, TestPid) end,
-    http_helper:start(Port, "/[:level1/[:level2/[:level3/[:level4]]]]", HandleFun).
+    {ok, _} = http_helper:start(0, "/[:level1/[:level2/[:level3/[:level4]]]]",
+                                          HandleFun),
+    Info = http_helper:info(),
+    proplists:get_value(port, Info).
```

Comment:
This is correct, thanks.

Would it be simpler and shorter to just use `ranch:get_port(http_helper_listener)`. This can be encapsulated inside `http_helper` module as `info`.

At this moment I don't see usage of any other data from the `Info` list. That's why I think YAGNI :) and KISS :)